exonum/src/sandbox/consensus/transactions.rs

```diff
@@ -554,3 +555,76 @@ fn request_txs_when_get_propose_or_prevote() {
 
     sandbox.add_time(Duration::from_millis(0));
 }
+
+const MAX_PROPOSE_TIMEOUT: Milliseconds = 200;
```

Comment:
Well, we can't put them into a function because they are shared between tests. I'll move the constants and the helper functions to the top of the file.